src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -841,10 +841,36 @@ public interface IConversionExpression : IHasOperatorMethodExpression
         /// Kind of conversion.
         /// </summary>
         ConversionKind ConversionKind { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an identity conversion.
+        /// </summary>
+        bool IsIdentity { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an implicit ('widening' in VB) conversion.
+        /// </summary>
+        bool IsImplicit { get; }
```

Comment:
I would be fine with picking either IsExplicit/IsImplicit or IsNarrowing/IsWidening.  Usually, it seems like for the shared codebases, we tend to follow C# naming.  i.e. we have ISwitchStatement not ISelectStatement.  But i'm not picky.